modules/osd-monitoring-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-configuring-the-monitoring-stack.adoc
+
+[id="configuring-the-monitoring-stack_{context}"]
+= Configuring the monitoring stack
+
+In {product-title}, you can configure the monitoring stack for user workloads using the `user-workload-monitoring-config` `ConfigMap` object. Config maps configure the Cluster Monitoring Operator (CMO), which in turn configures the components of the stack.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `dedicated-admin` role.
+* You have created the `user-workload-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `ConfigMap` object.
+.. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
+
+.. Add your configuration under `data/config.yaml` as a key-value pair `<component_name>:{nbsp}<component_configuration>`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    <component>:
+      <configuration_for_the_component>
+----
++
+Substitute `<component>` and `<configuration_for_the_component>` accordingly.
++
+The following example `ConfigMap` object configures a data retention period and minimum container resource requests for Prometheus. This relates to the Prometheus instance that monitors user-defined projects only:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus: <1>
+      retention: 24h <2>
+      resources:
+        requests:
+          cpu: 200m <3>
+          memory: 2Gi <4>
+----
+<1> Defines the Prometheus component and the subsequent lines define its configuration.
+<2> Configures a twenty-four hour data retention period for the Prometheus instance that monitors user-defined projects.
+<3> Defines a minimum resource request of 200 millicores for the Prometheus container.
+<4> Defines a minimum pod resource request of 2 GiB of memory for the Prometheus container.
++
+[NOTE]
+====
+The Prometheus config map component is called `prometheusK8s` in the `cluster-monitoring-config` `ConfigMap` object and `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object.
```

Comment:
I am afraid this note is not needed, since we don't mention `cluster-monitoring-config ConfigMap` at all in the doc

Reply:
Commented this out.